### PR TITLE
Updated the Primary Cooling's description

### PR DESCRIPTION
**Pull request by beccabunny**
**Content (Artwork / Missions / Jobs)**

## Summary
The current Primary Cooling's description is very stretched in game due to having such a long word as "power-hungry". This PR fixes this by changing that part of the description.
Before:
Screenshot (1298)
After:
Screenshot (1299)

### CODE REVIEW

File: data/ka'het outfits.txt [hunk 1]
```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although it requires a lot of energy, it is very effective, capable of upholding some Het by itself."
```
Comment (Pointedstick) [inline data/ka'het outfits.txt]: While you're changing this, I'm not sure that "upholding" is the right word to use here. Maybe "...it is capable of cooling entire Het shits by itself"

File: data/ka'het/ka'het outfits.txt [hunk 2]
```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although it requires a lot of energy, it is very effective, capable of upholding some Het by itself."
```
Comment (MasterOfGrey) [inline data/ka'het/ka'het outfits.txt]:
```suggestion
	description "This is the main cooler used by the Ka'het. Although it is energy intensive, it is very effective; capable of single- handedly cooling some Het ships."
```
Just a thought.

Comment (beccabunny) [reply data/ka'het/ka'het outfits.txt]: Added, thanks

File: data/ka'het/ka'het outfits.txt [hunk 3]
```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although it is energy intensive, it is very effective; capable of single- handedly cooling some Het ships."
```
Comment (Pointedstick) [inline data/ka'het/ka'het outfits.txt]: 1. "it is" is now used twice in this same sentence
2. changing the comma to a semicolon is incorrect
3. It's "single-handedy" (no space after the dash)

Comment (beccabunny) [reply data/ka'het/ka'het outfits.txt]: Changed again, how about this? 
"This is the main cooler used by the Ka'het. Although energy intensive, it is very effective, capable of single-handedly cooling some Het by itself."

Comment (Arrow2thekn33) [reply data/ka'het/ka'het outfits.txt]: That seems good to me.

File: data/ka'het/ka'het outfits.txt [hunk 4]
```diff
@@ -179,7 +179,7 @@ outfit "Ka'het Primary Cooling"
 	"outfit space" -14
 	"active cooling" 65
 	"cooling energy" 5.9
-	description "This is the main cooler used by the Ka'het. Although power-hungry, it is very effective, capable of upholding some Het by itself."
+	description "This is the main cooler used by the Ka'het. Although energy intensive, it is very effective, capable of single-handedly cooling some Het by itself."
```
Comment (tehhowch) [inline data/ka'het/ka'het outfits.txt]: >single-handedly cooling some Het by itself.

It's redundant to say both "single-handedly" and "by itself" -- I would drop the "by itself"

Comment (beccabunny) [reply data/ka'het/ka'het outfits.txt]: Dropped

**Pull request merged**